app/services/shelter_utilization.rb

```diff
@@ -33,8 +33,10 @@ def self.utilization_averages
     Census.group(:shelter_building).average(:count)
   end
 
-  def initialize
-    self.shelter_date = Date.today
+  def initialize(params = {})
+    period = params[:period_type] || :week
```

Comment:
I think period_type should default to day, since the default question is: "what is the utilization now"